File: julia/fizzbuzz1.jl

for i=0:100
    fzbz = false
    if i % 3 == 0
        fzbz = true
        print("Fizz")
    end
    if i % 5 == 0
        fzbz = true
        print("Buzz")
    end
    if !fzbz
        print(i)
    end
    println()
end

File: nim/fizzbuzz1.nim

for i in 0..100:
    var fzbz = false
    if i mod 3 == 0:
        fzbz = true
        stdout.write("Fizz")
    if i mod 5 == 0:
        fzbz = true
        stdout.write("Buzz")
    if not fzbz:
        stdout.write(i)
    stdout.write("\n")
